tools/crm_simulate.c

```diff
@@ -473,12 +476,15 @@ setup_input(const char *input, const char *output)
     cib_object = NULL;
 
     if (rc < 0) {
-        fprintf(stderr, "Could not create '%s': %s\n",
-                output, pcmk_strerror(rc));
-        crm_exit(CRM_EX_CANTCREAT);
+        rc = abs(rc);
```

Comment:
I'd still prefer legacy2rc even when we know it's errno, helps protect against any future changes in that function